x-pack/legacy/plugins/apm/e2e/run-e2e.sh

```diff
@@ -111,7 +118,7 @@ fi
 ##################################################
 echo "\n${bold}Waiting for Kibana to start...${normal}"
 echo "Note: you need to start Kibana manually. Find the instructions at the top."
-yarn wait-on -i 500 -w 500 http://localhost:$KIBANA_PORT > /dev/null
+yarn wait-on -i 500 -w 500 http://localhost:${KIBANA_PORT}/status > /dev/null
```

Comment:
See https://github.com/elastic/kibana/pull/48109/commits/8dd7213e32b5bcfa4acca54318f71a7c71201e57. We even tried to get 200 http code rather than 2xx:
- https://github.com/elastic/kibana/pull/48109/commits/3aeb1e1920faaf28953cfe2729f48900efdadd27

But it was reverted as it did not work as expected